.env.production

```diff
@@ -0,0 +1 @@
+S_PIPES_APP_TITLE=S-Pipes UI
```

Comment:
please change to `SPipes Editor` as this is targeted to users and not developers


note, we always use only variants: 
SPipes or s-pipes but never S-Pipes

Reply:
done